### PR TITLE
Add code lens and operations support to the template studio

### PR DESCRIPTION
**Pull request by m-vo**
This PR adds…

* code lens suppor to the template studio editors.
* a basic foundation for template studio operations.

There is also some refactoring, which is why both of these - otherwise independent - features are in one PR.

### CODE REVIEW

**Comment by m-vo** [comment]
I'm going to add the operations as well to this PR because then the refactoring is only done once.

File: core-bundle/contao/languages/en/template_studio.xlf [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                <source>Rename variant</source>
+            </trans-unit>
+            <trans-unit id="operation.rename_content_element_variant.1">
+                <source>Rename the "%s" variant template</source>
+            </trans-unit>
+            <trans-unit id="operation.create_frontend_module_variant.0">
+                <source>Create variant</source>
+            </trans-unit>
+            <trans-unit id="operation.create_frontend_module_variant.1">
+                <source>Create a new variant template for the "%s" frontend module</source>
```
Comment (leofeyer) [inline core-bundle/contao/languages/en/template_studio.xlf]:
```suggestion
                <source>Create a new variant template for the "%s" front end module</source>
```

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Comment by leofeyer** [comment]
Thank you @m-vo.

**Pull request merged**